pkgs/tools/misc/gif-for-cli/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, python3Packages, ffmpeg, zlib, libjpeg }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gif-for-cli";
+  version = "unstable-2018-12-15";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "gif-for-cli";
+    rev = "9696f25fea2e38499b7c248a3151030c3c68bb00";
+    sha256 = "1rj8wjfsabn27k1ds7a5fdqgf2r28zpz4lvhbzssjfj1yf0mfh7s";
+  };
+
+  checkInputs = [ python3Packages.coverage ];
+  buildInputs = [ ffmpeg zlib libjpeg ];
+  propagatedBuildInputs = [ python3Packages.pillow python3Packages.requests python3Packages.x256 ];
+
+  meta = with stdenv.lib; {
+    description = "Render gifs as ASCII art in your cli.";
```

Comment:
same here

Reply:
fixed, how about longDescriptions, I would expect that it can contain dots is that correct?

Reply:
Yeah that's fine. `longDescription` is completely optional, however.